script/script_hooks/destinations

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ -z "$DEFAULT_OSX_DESTINATION" ]; then
```

Comment:
I'd rather leave them because it's not some ironclad configuration like workspace or project, at some times I want to change it when I execute the command, but if I add it to the .env file these values will always override anything I setup in the CLI. This is useful when using the env matrix of travis to execute on diff OS in parallel.